### DISCUSSION TITLE
동적 메모리 할당기 | Jae’s Blog

### DISCUSSION THREAD

**Discussion by giscus** [jaenam615/jaenam615.github.io#6] [Announcements]
# 동적 메모리 할당기 | Jae’s Blog

Hydejack is a boutique Jekyll theme for hackers, nerds, and academics, with a focus on personal sites that are meant to impress.

https://jaenam615.github.io/computer_science/2024-02-07-allocator/

**Comment by moonmisae118** [comment]
동메할 즐감하고 가요~~
